main.py

```diff
@@ -0,0 +1,94 @@
+import click
```

Comment:
@taraspiotr I am on edge about naming and structure of stuff here because if we wanted to go with something django-like we would go with src -> app and main.py -> manage.py with pipeline_config.py -> settings.py . On the other hand we could simplify stuff and not have main.py fasade here at all (just explain in the instructions how to run it). 

Frankly I don't mind the django-like version one bit. It would make it easy to create a API service around it (if we wanted to) while at the same time making it easily readable for every django/python dev.

@taraspiotr @apyskir thoughts?